coral/src/domain/requests/requests-transformers.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestsWaitingForApproval } from "src/domain/requests/requests-types";
+import { KlawApiResponse } from "types/utils";
+
+const getRequestsWaitingForApprovalTransformer = (
```

Comment:
I'm missing tests for the transformer.

Reply:
It is :o

Reply:
[669c986](https://github.com/aiven/klaw/pull/731/commits/669c986076dcbbb14a712fda7db38a6f43779983)